spark-operator.yaml

```diff
@@ -24,17 +24,13 @@ pipeline:
     with:
       repository: https://github.com/GoogleCloudPlatform/spark-on-k8s-operator
       tag: spark-operator-chart-${{package.version}}
-      expected-commit: bd2eda216b79733c992861f257b28dad36bb4c62
+      expected-commit: 39b79d733b95d6ffe39a5ab50a65f92d03b62cc8
 
   - runs: |
       # Remediate GHSA-8mjg-8c8g-6h85 drop v1.19.5 to upgrade kubernetes version accordingly.
       go mod edit -dropreplace k8s.io/kubernetes
-
-  - uses: go/bump
-    with:
-      deps: golang.org/x/net@v0.17.0 google.golang.org/grpc@v1.56.3 golang.org/x/crypto@v0.17.0 k8s.io/kubernetes@v1.25.16 google.golang.org/protobuf@v1.33.0
-      replaces: golang.org/x/crypto=golang.org/x/crypto@v0.17.0
-      show-diff: true
+      go get k8s.io/kubernetes@v1.26.11
```

Comment:
This isn't changing the used version to be v1.26.11. There is a replace block that sets it to v1.19

Reply:
We have dropreplace in line 31

Reply:
- runs: |
      # Remediate GHSA-8mjg-8c8g-6h85 drop v1.19.5 to upgrade kubernetes version accordingly.
      go mod edit -dropreplace k8s.io/kubernetes
      # Note about the Found CVE here
      go mod edit -droprequire k8s.io/kubernetes
      go get k8s.io/kubernetes@v1.26.11